django_forest/settings.py

```diff
@@ -126,7 +128,7 @@
     'FOREST_URL': 'https://api.test.forestadmin.com',
     'APPLICATION_URL': 'http://localhost:8000',
     'FOREST_ENV_SECRET': 'foo',
-    'AUTH_SECRET': 'foo',
+    'AUTH_SECRET': base64.urlsafe_b64encode(uuid.uuid4().bytes),
```

Comment:
This will change each time the app starts right? Is this wanted/needed?

Reply:
This settings file is for the test app ;)
We can put whatever we want in fact.